WordPressAuthenticator/Extensions/CollectionType+AuthHelpers.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+
+/// Collection Helpers for Authenticator.
+///
+extension Collection {
+    /// Returns the element at the specified index if it is within bounds, otherwise nil.
+    subscript (safe index: Index) -> Element? {
```

Comment:
Maybe I missed something, but is this used by anything in this PR?

Reply:
Artifact from trying out a label and setting didSelectRowAt. Will delete!